src/praisonai-agents/praisonaiagents/memory/adapters/factories.py

```
@@ -371,6 +371,30 @@ def _sanitize_metadata(self, metadata: Dict) -> Dict:
     
+    def _reset_collection(self) -> None:
+        """Delete and recreate this adapter's collection, wiping stored memory.
+
+        Short-term and long-term share one collection in this adapter, so both
+        resets route here. Scoped to this adapter's own collection to avoid
+        wiping other collections in the shared persistent store.
+        """
+        try:
+            self.client.delete_collection(name=self.collection_name)
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)